system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -329,6 +331,13 @@ class CLdapConfig : implements ILdapConfig, public CInterface
         else
             m_ldap_secure_port = atoi(portbuf.str());
 
+        StringBuffer timeout;
+        cfg->getProp(".//@ldapTimeout()", timeout);
```

Comment:
also, consider using getpropInt... it allows you to specify default value thereby eliminating the next 4 lines